cocotb/drivers/__init__.py

```diff
@@ -29,6 +29,9 @@
 
 """Set of common driver base classes."""
 
+import socket
+import errno
+
```

Comment:
I don't agree with this change - there's no point increasing everyone's import time for a socket driver that probably no one uses.

Reply:
Rolled back.